index.html

```diff
@@ -69,6 +70,43 @@ <h4>Enterprise</h4>
           </div>
         </div>
       </section>
+
+      <section class="clients">
+        <div class="clients__testimonials">
+          <h4>Testimonials</h4>
+          <h2>What Clients Says About Us</h2>
+          <hr />
```

Comment:
can be deleted right? can't find the opening tag

Reply:
it is an horizontal break as in the mockup there is this kind of separator